experimental/MolecularBiology/WesternBlot/README.md

```diff
@@ -1 +1,108 @@
 
+# Protein Gel and Western Blot
+
+## reagents stored at the protein gel bench, JSCBB B450
+- 1.5M Tris pH 8.8
+- 1M Tris pH 6.8 
+- Loading buffer
+- Tris glycine SDS running buffer
+- Tris glycine methanol transfer buffer
+- 20X TBS
+
+## Pouring the gel
+#### Protein gels can be poured according to this spreadsheet (it is also hanging by the protein bench):
+*(Current Protocols in Molecular Biology)*
+
+<img width="525" alt="proteingel_pic" src="https://user-images.githubusercontent.com/64281908/80950465-25bc9600-8db3-11ea-9197-e1405807a539.png">
+
+### Pouring a gel *with* a stacking component (all recommended volumes are for a mini gel):
+> *1.5mm thick gels hold approximately 10mL liquid. This will be divided into approximately 7mL resolving gel and 3mL stacking gel. We typically load run 10% resolving gels.*
+1. Assemble the gel stand 
+  <br> *(optional) Fill with dI water to ensure there are no leaks, then pour out water and wick away as much as possible with a Kimwipe.
+  <br> (optional) Insert comb into the glass stand, and mark with a Sharpie where your wells end so you know how much resolving gel to add.*
+2. Make 10mL resolving gel solution, according to the table above. Both acrylamide solution and 10% ammonium persulfate (APS) are stored in the Midtown refrigerator (B455). _**(Be sure to use Tris pH 8.8!)**_
```

Comment:
A little more detail on how resolving gel solution is made.  It refers to the table above but maybe worth adding the table here.